packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -379,14 +379,14 @@ class CupertinoPickerDefaultSelectionOverlay extends StatelessWidget {
     const Radius radius = Radius.circular(_defaultSelectionOverlayRadius);
 
     return Container(
-      margin: EdgeInsets.only(
-        left: capLeftEdge ? _defaultSelectionOverlayHorizontalMargin : 0,
-        right: capRightEdge ? _defaultSelectionOverlayHorizontalMargin : 0,
+      margin: EdgeInsetsDirectional.only(
+        start: capLeftEdge ? _defaultSelectionOverlayHorizontalMargin : 0,
```

Comment:
I would recommend renaming the parameter to capStartEdge or capLeadingEdge.

Reply:
Thanks, you are right. 
done